Can not build on `macOS` - Missing dependencies

**What happened:**
Trying to build GL on macOS doesn't work anymore and fails:
```
% make kvm
Traceback (most recent call last):
  File "/Users/gyptazy/development/opensource/gardenlinux/bin/expand-flavours", line 4, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
make: *** No rule to make target `kvm'.  Stop.
```

**How to reproduce it**:
* Checkout a fresh git clone
* run `export GARDENLINUX_BUILD_CRE=docker`
* run `make kvm`
 
**Environment:**
```
Darwin mac-makebox.mg0.gyptazy.ch 22.3.0 Darwin Kernel Version 22.3.0: Mon Jan 30 20:39:35 PST 2023; root:xnu-8792.81.3~2/RELEASE_ARM64_T8103 arm64
```

| Name | Value |
| ------------------- | ----- |
| Garden Linux Branch | *main* |
| Container runtime (version)  | *Docker 4.9.1* | 
| Host OS of build machine  | *macOS* (arm64) |

--- a/README.md
+++ b/README.md
@@ -112,6 +112,9 @@ By default, Garden Linux uses [Podman](https://podman.io/) as container runtime 
 > # Install needed packages
 > brew install coreutils bash gnu-getopt gnu-sed gawk podman socat
 >
+> # Install Python dependencies via pip
+> pip3 install -r requirements.txt
+>
 > # Change to bash (Default: ZSH)
 > bash
 >

